tests/gate/test_verdict.py

```
@@ -157,15 +157,11 @@ def test_gate_contains_bootstrap_confidence_interval(
 
 
 def test_bootstrap_delta_ci_validation() -> None:
-    with pytest.raises(
-        ValueError, match="Both distributions must contain observations."
-    ):
+    with pytest.raises(ValueError, match="Both distributions must contain observations."):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)